snap/snapcraft.yaml

```diff
@@ -40,6 +40,8 @@ package-repositories:
     ppa: data-platform/python-pysyncobj
   - type: apt
     ppa: data-platform/pgbackrest-exporter
+  - type: apt
+    ppa: yevss/postgresql-14-timescaledb
```

Comment:
Copied here https://launchpad.net/~data-platform/+archive/ubuntu/postgresql-14-timescaledb
Please update snap config to build test snap.
We are still reviewing the package, please do NOT merge till the further notice. Tnx!

Reply:
@taurus-forever Updated PPA to fix failed build, I think it should be copied again

Reply:
I am lost here. Why are you using `yevss` PPA and now `data-platform` PPA directly?

Reply:
@taurus-forever I changed build details for `yevss` PPA and deployed it to test